eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala

```diff
@@ -1091,6 +1091,26 @@ class Channel(val nodeParams: NodeParams, val wallet: OnChainChannelFunder with
                   val commitments1 = d.commitments.add(signingSession1.commitment)
                   val d1 = d.copy(commitments = commitments1, spliceStatus = SpliceStatus.NoSplice)
                   log.info("publishing funding tx for channelId={} fundingTxId={}", d.channelId, signingSession1.fundingTx.sharedTx.txId)
+                  val fundingParams = signingSession1.fundingTx.fundingParams
+                  val sharedTx = signingSession1.fundingTx.sharedTx.tx
+                  if (fundingParams.localContribution > 0.sat) {
+                    Metrics.Splices.withTag(Tags.Origin, Tags.Origins.Local).withTag(Tags.SpliceType, Tags.SpliceTypes.SpliceIn).record(fundingParams.localContribution.toLong)
+                  }
+                  if (fundingParams.remoteContribution > 0.sat) {
+                    Metrics.Splices.withTag(Tags.Origin, Tags.Origins.Remote).withTag(Tags.SpliceType, Tags.SpliceTypes.SpliceIn).record(fundingParams.remoteContribution.toLong)
+                  }
+                  if (sharedTx.localOutputs.nonEmpty) {
+                    Metrics.Splices.withTag(Tags.Origin, Tags.Origins.Local).withTag(Tags.SpliceType, Tags.SpliceTypes.SpliceOut).record(sharedTx.localOutputs.map(_.amount).sum.toLong)
+                  }
+                  if (sharedTx.remoteOutputs.nonEmpty) {
+                    Metrics.Splices.withTag(Tags.Origin, Tags.Origins.Remote).withTag(Tags.SpliceType, Tags.SpliceTypes.SpliceOut).record(sharedTx.remoteOutputs.map(_.amount).sum.toLong)
+                  }
+                  if (fundingParams.localContribution < 0.sat && sharedTx.localOutputs.isEmpty) {
+                    Metrics.Splices.withTag(Tags.Origin, Tags.Origins.Local).withTag(Tags.SpliceType, Tags.SpliceTypes.SpliceCpfp).record(fundingParams.localContribution.toLong)
```

Comment:
I'm not sure we can record negative values? You probably want to take the absolute value of that one (same below)?

Reply:
Nice catch, done in b58a3bdb5e.